examples/cache/Dockerfile-frontenvoy

```diff
@@ -1,7 +1,10 @@
 FROM envoyproxy/envoy-dev:latest
 
-RUN apt-get update && apt-get -q install -y \
-    curl
+RUN apt-get update \
```

Comment:
almost certainly `curl` isnt needed and should just be removed - same in all the others

if it was required there is questionable benefit to cleaning up the apt dirs because these are not intended for distribution where size matters more (altho i guess there is an argument that it makes for a better example)

Reply:
Some examples call `curl` in container and write it in docs.

Reply:
ok - calling curl *in* the container would be exceptional and in that case (maybe) needed - but tbh that should not be happening in an envoy container

mentioning in the docs is (mostly at least) referring to curl in the host

Reply:
Yep, some examples enter container to exec `curl` like [this](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/front_proxy#step-5-enter-container-and-curl-admin).

Reply:
yuk!!!

i kinda get why that has been added (curl can be useful debugging a proxy from proxy's pov) but imho that should never have been added

either way - i think this is exceptional - my guess is that the other examples - eg cache - dont need it

Reply:
edit - cache isnt in pr - cors and csrf should not need curl tho

Reply:
updated.